### ISSUE TITLE
Invalid Integer Due Date

### ISSUE THREAD

**Issue by jmawebtech** [stripe/stripe-dotnet#1052]
Hi,

I am trying to create an invoice and I populated the due date field with a date. 

Here is my code:

```csharp
            StripeInvoiceService service = new StripeInvoiceService(m_UserName);
            StripeInvoice invoice = null;

            try
            {
                invoice = service.Create(customerId, inv, null);
            }
            catch (WebException ex)
            {
                using (var stream = ex.Response.GetResponseStream())
                using (var reader = new StreamReader(stream))
                {
                    string errorJson = reader.ReadToEnd();
                }
            }
            catch(Exception ex)
            {

            }
```

I receive an error saying the due date is an invalid integer.

**Comment by ob-stripe** [comment]
@jmawebtech How are you creating `inv`?

**Comment by jmawebtech** [comment]
Hi,

`          
string customerId = "cus_BlJzQ04x75cZma"; 
           StripeInvoiceCreateOptions invoice = new StripeInvoiceCreateOptions();
            invoice.Description = String.Format("QB Order Number: {0}", order.OrderNumber);
            invoice.DueDate = new DateTime(2017, 11, 13);
            invoice.Billing = StripeBilling.SendInvoice;
           StripeInvoiceService service = new StripeInvoiceService(key)l
           service.Create(customerId, invoice);
`

**Comment by ob-stripe** [comment]
@jmawebtech Thanks for the additional information. I was able to reproduce the issue and wrote a fix in #1055.

**Comment by jmawebtech** [comment]
Please let me know when this change is live, so I can download it. I assume it is not on NuGet. Should I just download the new source?

**Comment by ob-stripe** [comment]
The change has not been reviewed yet. If you're feeling adventurous, you can clone the `ob-fix-1052` branch.

**Comment by ob-stripe** [comment]
Released in v11.8.2.
